west-ci.yml

```diff
@@ -9,8 +9,8 @@ manifest:
   remotes:
     - name: zephyrproject-rtos
       url-base: https://github.com/zephyrproject-rtos
-    - name: secomind
-      url-base: git@github.com:secomind
+    - name: astarte-platform
+      url-base: https://github.com/astarte-platform
```

Comment:
Also, here I would keep it to the SSH format

Reply:
same as above